packages/react/src/design-system/index.ts

```diff
@@ -115,11 +115,11 @@ export const radius = {
 };
 
 export const iconSize = {
-  xs: "12px",
-  sm: "16px",
-  md: "24px",
-  lg: "32px",
-  xl: "48px",
+  xs: "12",
+  sm: "16",
+  md: "24",
+  lg: "32",
+  xl: "48",
 };
```

Comment:
should be `as const` I believe to get the stricter typing

Reply:
works just fine without it

<img width="300" alt="image" src="https://user-images.githubusercontent.com/22043396/225553230-e521f1d4-4207-44ea-8610-ed849155c938.png">

adding `as const` would make of `iconSize.xs` - `"12"` instead of `string` - which is not really required here

<img width="300" alt="image" src="https://user-images.githubusercontent.com/22043396/225553750-e7b19832-d14d-46d7-bb3f-8433a544a066.png">

Reply:
okay, fine to leave it then